src/app/chat/views/communities.nim

```diff
@@ -72,24 +83,30 @@ QtObject:
       self.activeCommunity.setCommunityItem(self.joinedCommunityList.getCommunityById(communityId))
       self.activeCommunity.triggerMemberUpdate()
 
+  proc populateChats(self: CommunitiesView, communities: seq[Community]): seq[Community] =
+    var i = 0
+    result = @[]
+    while (i<communities.len):
+      var community = communities[i]
```

Comment:
Why not use a `for`?
```nim
for community in communities:
```

Reply:
Because the community I need is a var (mutable). However when doing the for loop, the community is immutable.

Is there a way to achieve similar things with a for loop?

Reply:
Hmm, I thought is was mutable. Maybe `for var community in communities`? I'll check the docs

Reply:
```
for community in communities.mitems:
```

Reply:
Thanks @richard-ramos @jrainville it works!!! I just had to declare the sequence mutable manually in order to call this function now